modules/marsmediaBidAdapter.js

```diff
@@ -232,7 +232,19 @@ function MarsmediaAdapter() {
     };
     const bidString = JSON.stringify(bid);
     const encodedBuf = window.btoa(bidString);
-    utils.triggerPixel('https://ping-hqx-1.go2speed.media/notification/rtb/beacon/?bt=17&hb_j=' + encodedBuf, null);
+    utils.triggerPixel('https://ping-hqx-1.go2speed.media/notification/rtb/beacon/?bt=17&bid=3mhdom&hb_j=' + encodedBuf, null);
+  };
+
+  this.onTimeout = function (bid) {
+    const bidString = JSON.stringify(bid);
```

Comment:
Can these 3 lines be placed in a separate function? The function would then be called in `onBidWon` and be assigned to `onTimeout` and `onSetTargeting`.